game.py

```diff
@@ -20,7 +21,31 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+
+    wrong_guesses_list = []
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+
+        if user_input in snowman_word:
+            print("You guessed a letter that's in the word!")
+            #log correct letter guess
+            correct_letter_guess_statuses[user_input] = True
```

Comment:
A common rule of thumb is to perform the an operation and then after it is successful alert the user. This is so that there are no false positives sent to the user just in case of some failure in your code after the alert is since. With the `print` line being placed after it can only execute upon success of the prior lines.
```suggestion
            correct_letter_guess_statuses[user_input] = True
            print("You guessed a letter that's in the word!")
            # log correct letter guess
```